src/main/scala/battery/bot/core/CommandProcess.scala

```diff
@@ -58,6 +58,61 @@ class CommandProcess(persistenceService: PersistenceService, telegramClient: Tel
           .void
     }
   }
+
+  def sendSettings(sleepingTime: Int, wakeUpTime: Int, nightCharge: Boolean, result: Update): IO[Unit] = {
+    if (nightCharge) {
+      telegramClient
+        .sendMessage(
+          result.message.chat.id,
+          s"El usuario se levanta a las $sleepingTime, se acuesta a las $wakeUpTime y permite cargar durante la noche"
+        )
+        .void
+
+    } else {
+      telegramClient
+        .sendMessage(
+          result.message.chat.id,
+          s"El usuario se levanta a las $sleepingTime, se acuesta a las $wakeUpTime y no permite cargar durante la noche"
+        )
+        .void
+    }
+  }
+
+  def checkSettings(result: Update): IO[Unit] = {
+    for {
+      settingsList <- persistenceService.getUserSetting(result.message.from.username)
+      _            <- sendSettings(settingsList.sleepingTime, settingsList.wakeUpTime, settingsList.nightCharge, result)
+    } yield ()
+  }
+
+  def updateSettings(result: Update): IO[Unit] = {
+    val data = result.message.text.split(";").toList
+    data match {
+      case _ :: sleepingTime :: wakeUpTime :: nightCharge :: Nil =>
+        for {
+          _ <- persistenceService.updateUserSettings(
+            result.message.from.username,
+            sleepingTime.toInt,
+            wakeUpTime.toInt,
+            nightCharge.toBoolean
```

Comment:
If any of these conversions fails we should capture the errors and send an error response to the user